pkg/manager/client/discovery.go

```diff
@@ -30,15 +30,15 @@ func NewDiscoveryClient(ctx context.Context, config *rest.Config) (*DiscoveryCli
 	}, nil
 }
 
-func toObj(b []byte, namespace, groupVersion, kind string) interface{} {
+func toObj(b []byte, groupVersion, kind string) interface{} {
 
 	replaceString := strings.ReplaceAll(string(b), `"creationTimestamp":null`, `"creationTimestamp":"null"`)
 	replaceString = strings.ReplaceAll(replaceString, `\"creationTimestamp\":null`, `\"creationTimestamp\":\"null\"`)
 
 	finalString := strings.ReplaceAll(replaceString, `""`, `"null"`)
 	jsonParsed, err := gabs.ParseJSON([]byte(finalString))
 	if err != nil {
-		logrus.Errorf("Unable to parse json: %s, %s, %s", namespace, groupVersion, kind)
+		logrus.Errorf("Unable to parse json: %s, %s", groupVersion, kind)
```

Comment:
This part needs to be cleaned up/merged with other commits. This doesn't belong to the vendor change.